src/components/asset-list/RecyclerAssetList2/index.tsx

```diff
@@ -189,47 +189,58 @@ function NavbarOverlay({ accentColor, position }: { accentColor?: string; positi
   );
 
   return (
-    <Box
-      as={RNAnimated.View}
-      style={[
-        {
-          shadowColor: isDarkMode ? colors.shadowBlack : colors.rowDividerExtraLight,
-          shadowOffset: { width: 0, height: isDarkMode ? 4 : 1 },
-          // shadowOpacity: isDarkMode ? 0.4 : 0.04,
-          shadowRadius: isDarkMode ? 20 : 0,
-          zIndex: 1,
-        },
-        shadowOpacityStyle,
-      ]}
-    >
+    <>
       <Box
         as={RNAnimated.View}
-        background="surfacePrimary"
         style={[
           {
-            height: navbarHeight + insets.top + 24,
-            width: '100%',
-            position: 'absolute',
-            shadowColor: colors.shadowBlack,
-            shadowOffset: { width: 0, height: 1 },
+            shadowColor: isDarkMode ? colors.shadowBlack : colors.rowDividerExtraLight,
+            shadowOffset: { width: 0, height: isDarkMode ? 4 : 1 },
             // shadowOpacity: isDarkMode ? 0.4 : 0.04,
-            shadowRadius: 3,
-            top: navbarHeight + insets.top - 24,
+            shadowRadius: isDarkMode ? 20 : 0,
+            zIndex: 1,
+            height: navbarHeight + insets.top,
           },
-          animatedStyle,
+          shadowOpacityStyle,
         ]}
       >
         <Box
+          as={RNAnimated.View}
           background="surfacePrimary"
-          style={{
-            alignItems: 'center',
-            height: navbarHeight,
-            justifyContent: 'center',
-            top: insets.top + 24,
-          }}
-        />
+          style={[
+            {
+              height: navbarHeight + insets.top + 24,
+              width: '100%',
+              position: 'absolute',
+              shadowColor: colors.shadowBlack,
+              shadowOffset: { width: 0, height: 1 },
+              // shadowOpacity: isDarkMode ? 0.4 : 0.04,
+              shadowRadius: 3,
+              top: navbarHeight + insets.top - 24,
+            },
+            animatedStyle,
+          ]}
+        >
+          <Box
+            background="surfacePrimary"
+            style={{
+              alignItems: 'center',
+              height: navbarHeight,
+              justifyContent: 'center',
+              top: insets.top + 24,
+            }}
+          />
+        </Box>
       </Box>
-      <Box style={{ top: navbarHeight + insets.top, zIndex: 100 }}>
+      <Box
```

Comment:
Maestro cannot see views that overflow their parent, so I move the navbar outside of the view it used to be in.

Reply:
could you add PoW screen recordings for iOS and Android compared to before/after with these changes

Reply:
Ended up reverting this change since there was another change that fixed this (https://github.com/rainbow-me/rainbow/commit/eca35767325337ed3727fc19e6c22f43f0e4ce34).